Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM node:10 as builder
+
+WORKDIR /rest-on-couch-source
+COPY ./  ./
+
+RUN npm ci && npm run build && rm -rf node_modules
+
+FROM node:10
+
+ENV NODE_ENV production
+ENV REST_ON_COUCH_HOME_DIR /rest-on-couch
+
+WORKDIR /rest-on-couch-source
+COPY --from=builder /rest-on-couch-source /rest-on-couch-source
+RUN npm install -g pm2 && npm ci && rm -rf /root/.npm
```

Comment:
I don't really understand the point of having a multistage build in the Dockefile.

Why are you removing `node_modules` in the other image and redoing `npm ci` here?

Reply:
Here it's done with `NODE_ENV` set to "production". But you're right multistage is probably overkill in this case, especially now that `npm ci` already deletes `node_modules`. Feel free to improve the file.